### PR TITLE
add local counter vec

### PR DESCRIPTION
**Pull request by breeswish**
We already have `LocalHistogram`, `LocalHistogramVec` and `LocalCounter`. `LocalCounterVec` is missing. This PR adds it.

### CODE REVIEW

**Comment by siddontang** [comment]
This has a little difference with LocalHistogramVec which uses RefCell internally. I think we should unify them, maybe using RefCell here or removing RefCell in LocalHistogram. 

I prefer removing the RefCell if we find it is easy to update in TiKV.

/cc @overvenus

**Comment by siddontang** [comment]
Please add a PR description.

**Comment by breeswish** [comment]
@siddontang Done. The CI failure is wired. Could you take a look?

File: Cargo.toml [hunk 1]
```diff
@@ -35,7 +35,7 @@ path = "benches/benches.rs"
 [dependencies]
 protobuf = "1.4"
 quick-error = "0.2"
-clippy = {version = "^0", optional = true}
+clippy = {version = "*", optional = true}
```
Comment (siddontang) [inline Cargo.toml]: why change to asterisk here?

Comment (breeswish) [reply Cargo.toml]: changed according to [clippy doc](https://github.com/rust-lang-nursery/rust-clippy#optional-dependency), hoping to fix the clippy build failure but it seems that it doesn't work.

Comment (siddontang) [reply Cargo.toml]: I remember we used the asterisk before but some guys suggested us to use this.

/cc @overvenus

**Comment by breeswish** [comment]
@overvenus Done, PTAL

**Comment by overvenus** [comment]
LGTM, PTAL @siddontang

**Comment by siddontang** [comment]
@breeswish 

Please remember to update the LocalHistogram later.

File: Cargo.toml (hunk 1 above)
Comment (siddontang) [reply Cargo.toml]: Oh, I remember some guys suggested us not using the asterisk if we want to publish the library to the crate.

Comment (breeswish) [reply Cargo.toml]: @siddontang Ok, I reverted it.

**Comment by breeswish** [comment]
@siddontang Ok, it will be updated in another PR later.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**